### ISSUE TITLE
[RFC] Adding possibility to remove your extension

### ISSUE THREAD

**Issue by navelpluisje** [bolt/marketplace-site#63]
It would be nice if there is an option to remove your extension. This way you can remove your never finished extension. 
Effect will be a shorter list with better extensions

**Comment by GawainLynch** [comment]
Yeah, been on a TODO for a while … until then, if you do need anything removed, please let me know on Slack or IRC and I'll take care of it for you :smile:
